constants.go

```diff
@@ -0,0 +1,5 @@
+package vmcommon
+
+// VMTypeLen number of characters with VMType identifier in an address, these are the last 2 characters from the
+// initial identifier
+const VMTypeLen = 2
```

Comment:
VM should not care about this. Does not have knowledge about it.

Reply:
More info: `vmType` is an arbitrary byte string set by the node. In the future, it can be longer to include versioning for VMs in case of breaking changes between major versions. E.g. Arwen currently has type `0x05`, but Arwen 8 could have type `0x0508`. The argument parser can provide it without validation (it shouldn't be empty though).

Reply:
`AtArgumentsParser` - which is, actually, the **VM arguments parser** has the responsibility to **fully** parse both deploy arguments and contract call arguments. Thus the arguments parser should have knowledge on the length of the `vmType` in order to raise parsing errors in case of invalid length of this crucial deploy "argument".

In `vm-common` there are concepts shared by both the VM (all current and future) and the Node. `vmType` is a concept that VM is, actually, aware of. For example, see the constructor `NewArwenVM`.

Won't change. Reasonable design.